tools/packaging/kernel/build-kernel.sh

```diff
@@ -152,8 +152,8 @@ get_tee_kernel() {
 	local kernel_tarball="${version}.tar.gz"
 
 	# Depending on where we're getting the terball from it may have a
-	# different name, such as linux-${version}.tar.gz or simply
-	# ${version}.tar.gz.  Let's try both before failing.
+	# different name, such as linux-${version}.tar.gz, ${version}.tar.gz,
+	# or even v${version}.tar.gz.  Let's try all of them before failing.
 	curl --fail -L "${kernel_url}/linux-${kernel_tarball}" -o ${kernel_tarball} || curl --fail -OL "${kernel_url}/${kernel_tarball}"
```

Comment:
Should this have `|| curl --fail -OL "${kernel_url}/v${kernel_tarball}"` added, to try the or `v${version}.tar.gz` approach, or is the comment wrong?